ios/Client/Menu/EXDevMenuWindow.m

```diff
@@ -17,6 +17,8 @@ @implementation EXDevMenuWindow
 - (instancetype)init
 {
   if (self = [super init]) {
+    // Set it just above the LogBox so users can easily access it.
+    self.windowLevel = UIWindowLevelStatusBar - 0.5;
```

Comment:
i believe it's not necessary to subtract 0.5 from `UIWindowLevelStatusBar` - the dev menu shouldn't overlap with the status bar, and if that does happen it will still be rendered below the status bar (at least on ios 13+)